src/en/guide/plugins/creatingAndInstallingPlugins.adoc

```diff
@@ -90,6 +90,15 @@ Adds Quartz job scheduling features
 
 ==== Plugin Configuration
 
+A Grails plugin can be configured in one of the following files:
+
+- `grails-app/conf/plugin.yml`
+- `grails-app/conf/plugin.groovy`
+
+These files are the equivalent of `application.yml` or `application.groovy` in a Grails application.
```

Comment:
@gsartori This is not entirely correct. You can have an `application.yml` or `application.groovy` in a plugin project. These configs will just not make it into the consuming application. Examples of config settings that could be in a plugins `application.yml` file:
```yml
grails:
    profile: web-plugin
    codegen:
        defaultPackage: my.groovy.plugin
```

Reply:
What about:

A Grails plugin can be configured in one of the following files:

- `grails-app/conf/plugin.yml`
- `grails-app/conf/plugin.groovy`

These files will be included in the plugin package while the `application.yml` file is used when the plugin is run as an application. See [Notes on excluded Artefacts]().